src/Request.php

```diff
@@ -112,16 +112,103 @@ private function before_execute() {
 		}
 	}
 
+	/**
+	 * Checks authentication errors.
+	 *
+	 * False will mean there are no detected errors and
+	 * execution will continue.
+	 *
+	 * Anything else (true, WP_Error, thrown exception, etc) will prevent execution of the GraphQL
+	 * request.
+	 *
+	 * @return boolean
+	 *
+	 * @access protected
+	 *
+	 * @throws \Exception
+	 */
+	protected function has_authentication_errors() {
+
+		$authentication_errors = false;
+
+		if ( ! empty( $result ) ) {
```

Comment:
Where this variable is being set? I'm seeing it.

Reply:
oof. I did a refactor before opening the PR and didn't clean up after myself. 🤦‍♂ 

Thanks!